json/json.php

```diff
@@ -19,6 +19,7 @@ interface JsonSerializable
      * which is a value of any type other than a resource.
      * @since 5.4
      */
+    #[LanguageLevelTypeAware(['8.1'=>'mixed'], default: '')]
```

Comment:
Please also add class `LanguageLevelTypeAware` to imports and perform `docker-compose -f docker-compose.yml run php composer cs-fix -d /opt/project/phpstorm-stubs` to fix code-style related problems